app/src/main/java/com/example/vybe/Vibe.java

```diff
@@ -0,0 +1,6 @@
+package com.example.vybe;
+
+import java.io.Serializable;
+
+public class Vibe implements Serializable {
```

Comment:
This is supposed to be an EmotionalState, correct? (Referring to how this appears to be placed in the UML)

Reply:
Yes, Vibe = EmotionalState on the UML